jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -47,9 +47,15 @@ public class PathResource extends Resource
         .with("jrt")
         .build();
 
+    // The path object represented by this instance
     private final Path path;
+    // The as-requested URI for this path object
     private final URI uri;
-    private boolean targetResolved = false;
+    // True if this instance is an alias
+    private boolean alias = false;
+    // True to indicate that the alias resolution has been performed
+    private boolean aliasResolved = false;
```

Comment:
would a Boolean alias work better here?   null for unresolved and `Boolean.TRUE` or `Boolean.FALSE` once resolved.  That is correctly 3 states rather than using 3 of 4 possible states:
```suggestion
    // True/False if this instance is/isn't an alias. Null if alias resolution has not been performed.
    private Boolean alias;
```

Reply:
Using `Boolean` could work. (I'll give it a go)